main/models.py

```diff
@@ -251,9 +251,19 @@ def days_left(self) -> tuple[int, float] | None:
             The number of days and percentage worth of effort left, or None if there is
             no funding information.
         """
+        from .report import get_actual_chargeable_days
+
         if self.total_effort:
             left = sum([funding.effort_left for funding in self.funding_source.all()])
-            return left, round(left / self.total_effort * 100, 1)
+
+            # subtract days logged for the month so far
+            end_date = datetime.today().date()
+            start_date = end_date.replace(day=1)
+            additional_days = get_actual_chargeable_days(self, start_date, end_date)[0]
+            if additional_days:
+                left -= additional_days
+
+            return round(left), round(left / self.total_effort * 100, 1)
```

Comment:
@stephwills @dalonsoa here we are rounding off `left`, which will expect the number of days left to be an `int`. How about using `round(left, 1)` to account for the fractional days left too?